code-samples/calendar/python/calendar.py

```diff
@@ -0,0 +1,18 @@
+from nylas import APIClient
+
+client = APIClient(
+    "YOUR_CLIENT_ID",
+    "YOUR_CLIENT_SECRET",
+    access_token="ACCESS_TOKEN"
+)
```

Comment:
@JuanitaCathy  - On initial review - this looks like code for the Nylas v2 API, not v3.

v3 requires an api key. Let me see if I can provide access tomorrow for you to test this code out.